modules/cnv-example-ansible-playbook-creating-vms.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-automating-management-tasks.adoc
+
+[id="cnv-example-ansible-playbook-creating-vms_{context}"]
+= Example: Ansible Playbook for creating virtual machines
+
+You can use the `kubevirt_vm` Ansible Playbook to automate virtual machine
+creation.
+
+The following YAML file is an example of the `kubevirt_vm` playbook. It includes
+sample values that you must replace with your own information if you run the
+playbook.
+
+[source,yaml]
+----
+---
```

Comment:
Is the `---` necessary, or can the user assume they'll need to include their own stanza?

Reply:
IIUC, every Ansible Playbook starts with `---`, and this example is of the complete playbook (unlike the other example which is just a snippet).

Reply:
Yes, a whole playbook needs to start with the `---`.